examples/ppo_trainer/run_qwen2-7b_math_gsm8k_megatron.sh

```diff
@@ -0,0 +1,42 @@
+set -x
```

Comment:
could u later also add a section to https://github.com/volcengine/verl/blob/main/docs/experiment/ppo.rst , add a new table for MATH, and include the logs, command, and test score in next PR